Microsoft.Toolkit.Uwp.SampleApp/Controls/AadAuthControl.xaml.cs

```diff
@@ -0,0 +1,139 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System.Linq;
+using Microsoft.Toolkit.Uwp.UI.Controls.Graph;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Microsoft.Toolkit.Uwp.SampleApp.Controls
+{
+    public sealed partial class AadAuthControl : UserControl
+    {
+        public static readonly DependencyProperty IsShowAadMetaDataControlProperty = DependencyProperty.Register(
```

Comment:
Just bind the Visibility of your UserControl itself to your _aadAuthenticationManager.IsAuthenticated property.

Reply:
Done and committed.